### PR TITLE
Provide a way to get errors from Refresher

### PR DESCRIPTION
**Pull request by stellentus**
Were you planning to write test code for `Refresher`? If so, then I can add a test for this at that point. But I've tested it with `example/refresher` and it works.

### CODE REVIEW

File: refresher.go [hunk 1]
```diff
@@ -51,7 +55,16 @@ func (r *Refresher) launchIfNecessary(name string, value interface{}, f func(v i
...
 
 func (r *Refresher) ReadTag(name string, value interface{}) error {
-	r.launchIfNecessary(name, value, func(v interface{}) { r.plc.ReadTag(name, v) })
+	r.launchIfNecessary(name, value, func(v interface{}) {
+		err := r.plc.ReadTag(name, v)
+		if err != nil {
+			if r.ErrorCallback != nil {
+				r.ErrorCallback(err)
+			} else {
+				fmt.Println("WARNING: Unhandled plc.Refresher.ReadTag error:", err)
```
Comment (dijkstracula) [inline refresher.go:64]: Suggestion: migrate to using `log`

Comment (stellentus) [reply refresher.go]: Sure, that makes more sense. The `Println` is just a last resort reminder that errors weren't handled properly, but `log` is better.

Comment (stellentus) [reply refresher.go]: @dijkstracula Matthias suggested that I should delete the `Println` entirely. A user should be free to ignore errors without getting spammed by debug.

Comment (stellentus) [reply refresher.go]: ...so I've pushed another fixup with that change.

Comment (dijkstracula) [reply refresher.go]: That's reasonable. 👍

**Review by dijkstracula — APPROVED** [review]

**Comment by stellentus** [comment]
This is how I sneakily get people to accept emoji in my code. 😄

**Pull request merged**